src/lib/utils/functions/SanitizeHtmlUtils.ts

```diff
@@ -0,0 +1,109 @@
+import { IFilterXSSOptions } from 'xss'
+
+// PORTALS-1450: including 'style' in the allow-list will cause string values to come through, which crashes the app when used (because it uses jsx).
+export const xssOptions: IFilterXSSOptions = {
```

Comment:
Copied from SWC with a few additions per to match the existing `santize-html` config